src/components/form/RadioInput/RadioInput.js

```diff
@@ -0,0 +1,24 @@
+import { StyledRadioInput } from "./RadioInput.styled";
+
+export const RadioInput = ({ options, field, label, value, handleChange }) => {
+  return (
+    <StyledRadioInput>
+      <p>{label}</p>
+
+      {options.map((o) => (
```

Comment:
I would rather pass options as key/value object pairs
in that case e.g.: `[  { key: 1, value: 'cash' }, { key: 412, value: 'card' ]`  (prob have to pass additional e.g.: payment type)

why? we dont always have to generate multiple radio buttons (i know that for user experience in case of single radio button we should use checkbox instead, but anyways), but the idea is that we could have `paymentOptions` which comes from backend and changes regarding of user/use case/status/or whatever (like payment types can be different for some reason)

so first of all create something like this:
```
const paymentOptions = [
    { id: 5, type: "cash" },
    { id: 10, type: "card" },
  ];
 ```
 then remove this mapping from `RadioButton` component
 
 ```
   id,
  field,
  label,
  value,
  handleChange,
  isSelected,
}) => {
  return (
    <StyledRadioInput>
      <p>{label}</p>
      <input
        type="radio"
        id={id}
        name={label}
        value={id}
        onChange={handleChange}
        checked={isSelected === value}
      />
    </StyledRadioInput>
  );
  ```
  
  and then ussage of this should look like that
  
  ```
    <p>Payment method:</p>
        {paymentOptions.map(({ id, type }) => (
          <RadioInput
            key={id}
            value={type}
            label={type}
            isSelected={paymentType}   (it has to be fixed)
            handleChange={setPaymentType}
          />
        ))}
   ```
        
        our `RadioButton` is more generic now, we dont have to think about how to use it, just pass proper options, handleChange function etc.